### PR TITLE
Fix #619: fix display of attachment info

### LINKED ISSUE #619: Attachment info + delete attachment not shown
* Collection has attachment enabled/required
* Collection has JSONschema for `attachment`
* Record has attachment (image)
* Attachment fields are shown in form

--> But form does not show attachment info

screenshot from 2018-09-27 17-04-20
screenshot from 2018-09-27 17-04-43

### PR DESCRIPTION
**Pull request by leplatrem**
Fix #619 

screenshot from 2018-09-27 17-35-53

or when attachment is not required:
screenshot from 2018-09-27 17-41-25

### CODE REVIEW

File: src/components/record/RecordForm.js [hunk 1]
```diff
@@ -310,20 +310,20 @@ export default class RecordForm extends PureComponent<Props, State> {
...
-          {creation ? " create a" : " edit this"} record.
+          {isUpdate ? " edit this" : " create a"} record.
         </div>
       );
 
     return (
       <div>
         {alert}
-        {creation && (
+        {isUpdate && (
```
Comment (glasserc) [inline src/components/record/RecordForm.js]: Why not just `record && ...` ?

Comment (leplatrem) [reply src/components/record/RecordForm.js]: It's more obvious that it's only for update forms...

**Comment by leplatrem** [comment]
```
NODE_ENV=test node_modules/.bin/jest --setupTestFrameworkScriptFile=`pwd`/test/setup-jsdom.js --testRegex 'RecordAttributes_test.js'
 FAIL  test/components/RecordAttributes_test.js (11.03s)
  RecordAttributes component
    Attachment info shown
      ✕ should render the attachment info (88ms)
      ✕ should show a delete attachment button (2ms)

  ● RecordAttributes component › Attachment info shown › should render the attachment info

    Error: Uncaught [Error: Invalid arguments]

````

**Comment by leplatrem** [comment]
`attachment.size` was undefined in the tests, and the `<AttachmentInfo/>` was failing while calling `filesize()`.

**Pull request merged**